InstructionTuning/README.md

```diff
@@ -2,7 +2,17 @@
 
 Instruction tuning is the process of further training LLMs on a dataset consisting of (instruction, output) pairs in a supervised fashion, which bridges the gap between the next-word prediction objective of LLMs and the users' objective of having LLMs adhere to human instructions. This implementation deploys a Ray cluster for the task.
 
-## Deploy Instruction Tuning Service
+# Table of contents
+
+1. [Architecture](#Architecture)
+2. [Deployment Options](#Deployment-Options)
+3. [Instruction Tuning Service Usage](#Instruction-Tuning-Service-Usage)
```

Comment:
It would be better to include the detailed Instruction Tuning Service Usage section under each deployment option, not at the top layer.

Reply:
Since usage is the same for all deployments now, it's for avoiding duplication.